<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QxrdCenterFinderDialog</class>
 <widget class="QGroupBox" name="QxrdCenterFinderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Centering</string>
  </property>
  <property name="title">
   <string>Centering</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QGridLayout" name="gridLayout_8">
        <item row="0" column="0">
         <widget class="QToolButton" name="m_CenterMoveUpLeft">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-up-left.png</normaloff>:/images/images/go-up-left.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QToolButton" name="m_CenterMoveUp">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-up.png</normaloff>:/images/images/go-up.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QToolButton" name="m_CenterMoveUpRight">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-up-right.png</normaloff>:/images/images/go-up-right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="m_CenterMoveLeft">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-left.png</normaloff>:/images/images/go-left.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="qxrdresources.qrc">:/images/images/color-wheel.png</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="m_CenterMoveRight">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-right.png</normaloff>:/images/images/go-right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="m_CenterMoveDownLeft">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-down-left.png</normaloff>:/images/images/go-down-left.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QToolButton" name="m_CenterMoveDown">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-down.png</normaloff>:/images/images/go-down.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="m_CenterMoveDownRight">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="qxrdresources.qrc">
            <normaloff>:/images/images/go-down-right.png</normaloff>:/images/images/go-down-right.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>X Center:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="m_CenterX"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Y Center:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="m_CenterY"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Step Size:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="m_CenterStep"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="qxrdresources.qrc"/>
  <include location="qxrdresources.qrc"/>
 </resources>
 <connections/>
</ui>
